Cargo.toml

```diff
@@ -29,6 +29,7 @@ tokio = { version = "1.0", features = ["macros", "rt-multi-thread"], optional =
 uuid = { version = "1.1.2", features = ["serde", "v4"], optional = true }
 bincode = { version = "1.3.3", optional = true }
 futures-util = "0.3.21"
+base64-url = "1.4.13"
```

Comment:
if we don't want to introduce a new crate (we already use `base64` above), we can use the existing crate with:

```rust
let b64_url = base64::encode_config(b"hello internet~", base64::URL_SAFE);
```

Reply:
The `base64` crate is much more established (just looking at github stars, contributors, and crates.io downloads count). So my preference is to switch the calls to `base64::encode_config()`, and remove the new dependency.

BTW we should use config `base64::URL_SAFE_NO_PAD` and not like I wrote above.